test/helpers/test_executions_helper_test.rb

```diff
@@ -112,6 +112,60 @@ def test_get_select_history_message
     assert_includes msg, '(passing)'
   end
 
+  def test_population_errors
+    errors = []
```

Comment:
@ljades would it make sense to just make one errors array at put it in the setup for this test class?